### DISCUSSION TITLE
How to reset the Item after search?

### DISCUSSION THREAD

**Discussion by mripqi** [azeezat/react-native-select#95] [Q&A] [answered]
I m using the DropdownSelect Currency in the examples, after i search using 'a' or 'o' in the search box it will update the item in the list but when i delete it, the item still wont go back to the initial Item.
i think it's because in use-search.ts it will only run when there is a searchValue
`  useEffect(() => {
    if (searchValue) {
      onSearch(searchValue);
    }
  }, [onSearch, searchValue]);`
  
  typing space will reset the item, but i want it to reset when the user delete the searchValue. Is there anyway to achieve this?

**Comment by kofiarkoh** [comment, marked as answer]
@mripqi , thanks for sharing this. I have been able to reproduce this and I think  this will require some minor update....

**Reply by kofiarkoh** [reply]
@azeezat ,  please correct me if i am wrong

**Reply by azeezat** [reply]
@kofiarkoh Thanks, have you been able to resolve this? otherwise, I can take a look when I'm free.

**Reply by kofiarkoh** [reply]
@azeezat , sure i will work on it.
expect a PR soon
